docs/release-notes/release-notes-0.18.0.md

```diff
@@ -246,6 +246,11 @@ bitcoin peers' feefilter values into account](https://github.com/lightningnetwor
   change. And the fee estimation is correct even if no change output is
   required.
 
+* Some new [RPCs for managing SCID 
```

Comment:
Let's move this to a new and empty 0.18.1 file (we'll add the rest of the skeleton before merging the staging branch into master).